repldex/backend/static/main.css

```diff
@@ -1,4 +1,4 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
Why switch to Google fonts? We could even put the font as a static file in Repldex so it doesn't have to preconnect.

Reply:
I switched to Google Fonts because you guys put way too many requests to the other URL